integration-cli/docker_api_inspect_test.go

```diff
@@ -115,8 +115,8 @@ func (s *DockerSuite) TestInspectAPIImageResponse(c *check.C) {
 	c.Assert(err, checker.IsNil)
 
 	c.Assert(imageJSON.RepoTags, checker.HasLen, 2)
-	c.Assert(stringutils.InSlice(imageJSON.RepoTags, "busybox:latest"), checker.Equals, true)
-	c.Assert(stringutils.InSlice(imageJSON.RepoTags, "busybox:mytag"), checker.Equals, true)
+	c.Assert(testutil.InSlice(imageJSON.RepoTags, "busybox:latest"), checker.Equals, true)
+	c.Assert(testutil.InSlice(imageJSON.RepoTags, "busybox:mytag"), checker.Equals, true)
```

Comment:
looks like here it doesn't have to be case-insensitive as well

Reply:
definitely not.